pkg/solana/chain.go

```diff
@@ -291,6 +293,9 @@ func newChain(id string, cfg *config.TOMLConfig, ks core.Keystore, lggr logger.L
 
 		ch.multiNode = multiNode
 		ch.txSender = txSender
+		ch.multiClient = client.NewMultiClient(func() (client.ReaderWriter, error) {
+			return ch.multiNode.SelectRPC()
+		})
```

Comment:
Isn't this redundant with the prior initialization?
```
ch.multiClient = client.NewMultiClient(ch.getClient)
```
Either way, if the multinode is enabled the same `SelectRPC()` function will be called, right?
```
// getClient returns a client, randomly selecting one from available and valid nodes
// If multinode is enabled, it will return a client using the multinode selection instead.
func (c *chain) getClient() (client.ReaderWriter, error) {
	if c.cfg.MultiNode.Enabled() {
		return c.multiNode.SelectRPC()
	}
```

Reply:
Actually I just re-read @DylanTinianov's comment on my PR and saw he's proposing to get rid of the MultiNode path from `c.getClient`.  If we do that instead, then I think this makes more sense.

Reply:
We could either do that in this PR, or in my PR